compiler/cpp/src/thrift/generate/t_swift_generator.cc

```diff
@@ -947,7 +970,7 @@ void t_swift_generator::telemetry_dictionary_value(ofstream& out, t_type* type,
 
     out << "] = ";
 
-    telemetry_dictionary_value(out, tmap->get_val_type(), "value");
+    telemetry_dictionary_value(out, tmap->get_val_type(), "value", "");
```

Comment:
Should this be `nil`?

Reply:
I changed to pass in `pii_kind`. I figure if someone wants to set the PIIKind of the telemetry dictionary they should be able to do that